### ISSUE TITLE
How is the running speed?

### ISSUE THREAD

**Issue by 2020zyc** [zysite/biaffine-parser#5]
hi, how is the decode speed? Any evaluation? Thx.

**Comment by zysite** [comment]
I didn't evaluate the parsing speed precisely like the paper, so it may not be able to compare with it correctly.
The time for training a epoch is about 4m50s, and it takes about 1m12s to evaluate the three datasets (43948 sents). I didn't estimate the time of decoding individually.

**Comment by 2020zyc** [comment]
ok, thx and nice code.

**Comment by zysite** [comment]
Thanks :)

**Comment by zysite** [comment]
Recently I implemented the bucket mechanism adopted in tensorflow, which has made the speed twice as fast as I said before, but may result in a slightly loss of performance (95.79 & 94.08, still better than the official code).
The implementation is still not perfect, but may give you some inspiration.
